lib/fastlane_core/helper.rb

```diff
@@ -102,6 +102,19 @@ def self.iterm?
       !!ENV["ITERM_SESSION_ID"]
     end
 
+    def self.openssl_version
+      return @openssl_version if @openssl_version
+
+      begin
+        output = `openssl version`
```

Comment:
MMM I wasn't aware of this option. I always installed it uing brew.

http://stackoverflow.com/questions/14679878/openssl-error-got-when-install-ruby-2-0-0-via-rvm-and-rbenv

I will have to test that. Intuitively, it should work, as rvm modifies the PATH so the `openssl version` call should represent what rvm sees as well. But better to test more... I'll need some time to set this up.